front50-test/src/main/groovy/com/netflix/spinnaker/front50/utils/CassandraTestHelper.groovy

```diff
@@ -0,0 +1,38 @@
+package com.netflix.spinnaker.front50.utils
+
+import com.netflix.astyanax.Keyspace
+import com.netflix.astyanax.connectionpool.impl.ConnectionPoolConfigurationImpl
+import com.netflix.spinnaker.kork.astyanax.AstyanaxComponents
+
+
+public class CassandraTestHelper {
+  AstyanaxComponents.EmbeddedCassandraRunner runner
+  Keyspace keyspace
+
```

Comment:
This is moved into `front50-test` and generified a bit such that it could be used in `front50-web` tests (see `PipelineControllerTCK` for an example).